docs/tutorials/educators/intro.ipynb

```diff
@@ -70,7 +70,7 @@
    "source": [
```

Comment:
<p>Defining LineQubits was intentional in the solution so that we can introduce the learners to different types of qubits. </p><p>They would have attempted NamedQubits in the exercise above and while discussing the solution, we'd point out that you can achieve the same task more easily by using LineQubit.range(4). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3981/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Sounds good! I'll add this sentiment in the comment then - that NamedQubits are similarly okay here!